srtcore/epoll.cpp

```diff
@@ -369,6 +313,49 @@ int CEPoll::update_ssock(const int eid, const SYSSOCKET& s, const int* events)
    return 0;
 }
 
+int CEPoll::uwait(const int eid, SRT_EPOLL_EVENT* fdsSet, int fdsSize, int64_t msTimeOut, bool edgeMode)
+{
+    int64_t entertime = CTimer::getTime();
+    if (!fdsSet && fdsSize)
```

Comment:
Does this mean that zero size is allowed if you pass NULL fdsSet? What about negative size, stating that the type is `int`? Is nonempty array officially allowed and what sense does it make?